libcudacxx/include/cuda/std/detail/libcxx/include/bitset

```diff
@@ -129,932 +137,889 @@ template <size_t N> struct hash<std::bitset<N>>;
 #  pragma system_header
 #endif // no system header
 
-_LIBCUDACXX_PUSH_MACROS
-#include <__undef_macros>
+#include <cuda/std/__algorithm/count.h>
+#include <cuda/std/__algorithm/fill.h>
+#include <cuda/std/__algorithm/find.h>
+#include <cuda/std/detail/libcxx/include/__assert> // all public C++ headers provide the assertion handler
+#include <cuda/std/detail/libcxx/include/__bit_reference>
+#include <cuda/std/__functional/hash.h>
+#include <cuda/std/__functional/unary_function.h>
+#include <cuda/std/detail/libcxx/include/__string>
+#include <cuda/std/__type_traits/is_char_like_type.h>
+#include <cuda/std/detail/libcxx/include/climits>
+#include <cuda/std/detail/libcxx/include/cstddef>
+#include <cuda/std/detail/libcxx/include/stdexcept>
+#if defined(_LIBCUDACXX_HAS_STRING_VIEW)
+#  include <cuda/std/detail/libcxx/include/string_view>
+#endif
+#include <cuda/std/detail/libcxx/include/version>
```

Comment:
I believe we can drop all the `detail/libcxx/include/` if it pertains released headers, which they should be